homeassistant/components/airvisual/sensor.py

```diff
@@ -271,17 +269,7 @@ def device_info(self):
                 f'{self.coordinator.data["status"]["app_version"]}'
```

Comment:
```suggestion
                f'{coordinator.data["status"]["app_version"]}'
```

Reply:
I can't see this entire dict while reviewing, but you can remove any other references to `self` that I missed.